.gitignore

```diff
@@ -5,3 +5,4 @@
 *~
 .project
 .pydevproject
+wsgi/media/**
```

Comment:
Ohh them try to ignore particularly that only..because the file lookup is at directory `media/photos/<file_name>.jpg`